src/main/webapp/index.jsp

```diff
@@ -38,7 +38,7 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>CodeU Chat App</h1>
-      <h2>~ Created by Lucie and Kevin! ~</h2>
+      <h2>~ Created by Lucie, Kevin and Lauren! ~</h2>
```

Comment:
This change is already "merged" into master right? 
See master branch https://github.com/lucieleblanc/CodeU-Summer-2018/blob/master/src/main/webapp/index.jsp

So this is a duplicate pull request? It can be dropped?

Reply:
I agree, this seems like it may be a duplicate.